src/libs/Permissions.ts

```diff
@@ -0,0 +1,69 @@
+import CONST from '../CONST';
+
+function canUseAllBetas(betas: string[]): boolean {
```

Comment:
@BartoszGrajdek Is it possible to use `OnyxTypes.Beta[]` instead of `string[]` for all the functions here?

Reply:
Done!